app/src/test/java/com/onboarding/parkingsystemjava/utils/ConstUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.onboarding.parkingsystemjava.utils;
+
+public class ConstUtils {
```

Comment:
You should not use this class. each test class should have all the necessary info